WtWSMS/common/landed_titles/BP_e_byzantium.txt

```diff
@@ -11991,17 +13325,13 @@ e_byzantium = {
 			c_zherkovo = {
 				color = { 50 75 85 }
 				cultural_names = {
-					#Byzantine Cultures
-					name_list_greek = cn_hyperperakion
 					#Turkic Cultures
 					name_list_turkish = cn_kircaali
 				}
 				
 				b_zherkovo = {
 					province = 3618
 					cultural_names = {
-						#Byzantine Cultures
-						name_list_greek = cn_hyperperakion
```

Comment:
Same here?

Reply:
> Same here?

15 km from the site is too far away IMO.

Reply:
Still within the same province, better to refer to another settlement than something which didn't exist

Reply:
> Still within the same province, better to refer to another settlement than something which didn't exist

Reverted.